meta-gpsnavi-agl/recipes-navigation/navi/navidata_0.0.7.bb

```diff
@@ -0,0 +1,27 @@
+DESCRIPTION = "Navigation Open Source Project Navi Data."
+HOMEPAGE = "http://agl.wismobi.com/"
+LICENSE="GPLv2 | COMMERCIAL"
```

Comment:
This data is based on OpenStreetMap, so the license is Open Database License ("ODbL-1.0" in Yocto common license name). Could you fix this?